frontend/src/pages/VersionTree/index.tsx

```diff
@@ -61,28 +62,98 @@ export default class VersionTree extends React.Component<{}, any> {
   constructor(props) {
     super(props);
     this.state = {
-      gitTreeData: []
+      gitTreeData: [],
+      modelData: []
     };
   };
-  public componentDidMount() {}
+  public async componentDidMount() {
+  }
 
+  /**
+   * TODO: display optimized formats(variant) of models
+   * @param gitgraph GitgraphUserApi Object
+   */
   public async generateGitTree(gitgraph) {
-    const res = await axios.get(config.modelURL);
-    res.data.reverse().sort((a, b) => new Date(b.create_time) - new Date(a.create_time))
-    this.setState({gitTreeData: generateGitData(res.data)})
+    let res = await axios.get(config.modelURL);
+    let modelList = res.data;
+    this.setState({
+      gitTreeData: generateGitData(modelList),
+      modelData: modelList.reverse().filter(model => ['PYTORCH', 'None'].indexOf(model.engine) >= 0)
+    })
     gitgraph.import(this.state.gitTreeData);
-    console.log(this.state.gitTreeData)
+    // gitgraph.import(mockData)
   }
 
+  const columns = [
+    {
+      title: 'Tags',
+      dataIndex: 'name',
+      key: 'tag',
+      render: (name, record) =>
+        <div>
+          {record.parent_model_id ? '' : <Tag color='gold'>Original</Tag>}
+          <Tag>{record.dataset}</Tag>
+        </div>,
+    },
+    {
+      title: 'version',
+      dataIndex: 'version',
+      key: 'version',
+      render: version => <Tag color='green'>v{version}</Tag>
+    },
+    {
+      title: 'Creator',
+      dataIndex: 'creator',
+      key: 'creator',
+      render: creator =>
+        <div>
+          <Space size="large">
+            <Avatar size={48} src={`https://avatars.dicebear.com/4.5/api/identicon/:${creator}.svg`} />
+            <span style={{ whiteSpace: 'nowrap', overflow: 'hidden', textOverflow: 'ellipsis', width: 100, display: 'inline-block' }}>
+              {creator}
+            </span>
+          </Space>
+        </div>
+    },
+    {
+      title: 'Create time',
+      key: 'create_time',
+      dataIndex: 'create_time',
+      render: create_time => moment(create_time).fromNow()
+    },
+    {
+      title: 'Action',
+      dataIndex: 'id',
+      key: 'action',
+      render: (id) => (
+        <Link to={`/visualizer/${id}`}>
+          <EditOutlined style={{ fontSize: 30 }} />
+        </Link>
+      ),
+    },
+  ];
   public render() {
     return (
-      <Gitgraph
-        options={{ template: gitGraphOption }}
-      >
-        {(gitgraph) => {
-          gitgraph.import(mockData);
-        }}
-      </Gitgraph>
+      <Row>
+        <Col span={2} offset={2}>
+          <Gitgraph
+            options={{ template: gitGraphOption, orientation: Orientation.VerticalReverse }}
```

Comment:
I think just use the normal direction (the newest at the top, and the oldest at the bottom). Since the new model is more likely to be used.